lagtraj/forcings/conversion/targets/kpt.py

```diff
@@ -9,7 +9,11 @@
 import xarray as xr
 
 from ....domain.sources.era5.constants import rg
-from ....utils.interpolation.methods import central_estimate, steffen_1d_no_ep_time
+from ....utils.interpolation.methods import (
+    central_estimate,
+    cos_transition,
```

Comment:
Needed for the both the inversion nudging and the correction to wind at high levels.